google-cloud-core/README.md

```diff
@@ -6,7 +6,14 @@ This library contains shared types, such as error classes, for the google-cloud
 
 ## Supported Ruby Versions
 
-This library is supported on Ruby 2.0+.
+This library is currently supported on Ruby 2.0+.
+
+However, Ruby 2.3 or later is strongly recommended, as earlier releases are
+or are nearing end-of-life. After June 1, 2018, Google will provide official
+support only for Ruby versions that are considered current and supported by
+Ruby-core (that is, Ruby versions that are either in normal maintenance or in
```

Comment:
"Ruby-core" -> "Ruby Core" (Or is it "Ruby-Core"?)

Reply:
That's a good question. I'm not sure there's a real "official" entity here. I think people generally refer to it as the "Ruby core team"?

Reply:
https://www.ruby-lang.org/en/community/ruby-core/ is titled "Ruby Core" but the [mailing list](https://www.ruby-lang.org/en/community/mailing-lists/) is "Ruby-Core". Beats me, but I think capitalizing "Core" is ok.

Reply:
Changed to "Ruby Core"